include/zephyr/bluetooth/gatt.h

```diff
@@ -1488,6 +1488,22 @@ bool bt_gatt_is_subscribed(struct bt_conn *conn,
  */
 uint16_t bt_gatt_get_mtu(struct bt_conn *conn);
 
+/** @brief Get UATT MTU for a connection
+ *
+ *  Get UATT connection MTU.
+ *
+ * The ATT_MTU defines the largest size of an ATT PDU, encompassing the ATT
+ * opcode, additional fields, and any attribute value payload. Consequently,
+ * the maximum size of a value payload is less and varies based on the type
+ * of ATT PDU. For example, in an ATT_HANDLE_VALUE_NTF PDU, the Attribute Value
+ * field can contain up to ATT_MTU - 3 octets.
+ *
+ *  @param conn Connection object.
+ *
+ *  @return MTU in bytes
```

Comment:
It looks like the implementation can return `0` in some cases. We should document this.

Reply:
For example:
```
@retval 0 If @p conn does not have an UATT ATT_MTU, for example because it's disconnected.
@retval >0 The UATT ATT_MTU at this moment.
```

Reply:
Sounds good. Documented this retval explicitly